redcap_bridge/test_redcap/test_cli.py

```diff
@@ -50,3 +50,10 @@ def test_download(initialize_test_dir):
                             stdout=subprocess.PIPE)
     assert 'error' not in str(result.stdout)
     assert pathlib.Path(output_file).exists()
+
+    # download with Elabftw server
+    result = subprocess.run(['RedCapBridge', 'download', SERVER_CONFIG_YAML, '--server', 'elabftw', '232'],
```

Comment:
you are missing the output files argument here, that should still be mandatory, no?

Reply:
I don't put the output because the download function is really simple, my idea was to add the output after